.gitignore

```diff
@@ -1 +1,4 @@
 Godeps/_workspace
+.DS_Store
+testpop
```

Comment:
Ah. Probably I was changing directory to build it. Can we remove the `scripts/testpop/.gitignore` then? It's better off in the `.gitignore` if that's the case.